appcheck/firebase-appcheck-debug-testing/gradle.properties

```diff
@@ -1,2 +1,2 @@
-version=16.1.3
+version=17.0.0
```

Comment:
Should the SafetyNet and PlayIntegrity App Check SDKs also be bumped? I think we were generally trying to keep the version numbers aligned.

Reply:
They seemed behind the last minor so I wasn't sure, bumped them as well